src/components/layout/footer.js

```diff
@@ -242,44 +243,73 @@ const mobile_accordion_header = {
     backgroundColor: 'var(--color-grey-25)',
     boxShadow: 'none',
 }
+
 const mobile_accordion_header_about = Object.assign({}, mobile_accordion_header)
-const SocialWrapperComponent = () => {
+
+const SocialWrapperComponent = ({ is_career_page }) => {
+    const alt_string = (is_career_page ? 'career' : '') + ' icon link'
+    const accounts = [
+        {
+            index: 1,
+            link: is_career_page
+                ? 'https://www.facebook.com/derivcareers'
+                : 'https://www.facebook.com/derivdotcom/',
+            image: Facebook,
+            image_alt: `facebook ${alt_string}`,
+        },
+        {
+            index: 2,
+            link: 'https://twitter.com/derivdotcom/',
+            image: Twitter,
+            image_alt: `twitter ${alt_string}`,
+        },
+        {
+            index: 3,
+            link: is_career_page
+                ? 'https://www.instagram.com/derivcareers/'
+                : 'https://www.instagram.com/deriv_official/',
+            image: Instagram,
+            image_alt: `instagram ${alt_string}`,
+        },
+        {
+            index: 4,
+            link: 'https://www.linkedin.com/company/derivdotcom/',
+            image: Linkedin,
+            image_alt: `linkedin ${alt_string}`,
+        },
+    ]
+
     return (
-        <SocialWrapper>
-            <LocalizedLink
-                external="true"
-                to="https://www.facebook.com/derivdotcom/"
-                target="_blank"
-                rel="noopener noreferrer"
-            >
-                <img src={Facebook} alt="facebook" width="41" height="41" />
-            </LocalizedLink>
-            <LocalizedLink
-                external="true"
-                to="https://twitter.com/derivdotcom"
-                target="_blank"
-                rel="noopener noreferrer"
-            >
-                <img src={Twitter} alt="twitter" width="41" height="41" />
-            </LocalizedLink>
-            <LocalizedLink
-                external="true"
-                to="https://www.instagram.com/deriv_official/"
-                target="_blank"
-                rel="noopener noreferrer"
-            >
-                <img src={Instagram} alt="instagram" width="41" height="41" />
-            </LocalizedLink>
+        <SocialMediaComponent
+            social_accounts={
+                is_career_page ? accounts.filter((account) => account.index != 2) : accounts
+            }
+        />
+    )
```

Comment:
I guess it's better to have the `facebook - linkedin - instagram` in the base array. Since the condition controls `twitter` you can use [splice](https://stackoverflow.com/questions/586182/how-to-insert-an-item-into-an-array-at-a-specific-index-javascript) to insert it into the 1st index when the condition is met.

This is to provide more clear code.
it will be smth like that:
```
let arr = ...
if (is_career_page) {
    arr.splice(...)
}
return <SocialMediaComponent social_accounts={accounts} />
```